### PR TITLE
New package: haruna-0.8.0

### PR DESCRIPTION
**Pull request by dragon-friend**
#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): YES

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/haruna/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haruna'
+
+pkgname=haruna
+version=0.8.0
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
+makedepends="mpv-devel ffmpeg-devel qt5 qt5-dbus qt5-declarative-devel qt5-quickcontrols2-devel extra-cmake-modules kconfig-devel kcoreaddons-devel kfilemetadata5-devel ki18n-devel kiconthemes-devel kio-devel kirigami2-devel kxmlgui-devel kdoctools-devel breeze AppStream gettext-devel"
+depends="breeze-icons breeze qqc2-desktop-style youtube-dl"
+short_desc="Haruna is an open source video player built with Qt/QML and libmpv"
```
Comment (classabbyamp) [inline srcpkgs/haruna/template:10]:
```suggestion
short_desc="Open source video player built with Qt/QML and libmpv"
```
having the first bit seems redundant

File: srcpkgs/haruna/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haruna'
+
+pkgname=haruna
+version=0.8.0
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
+makedepends="mpv-devel ffmpeg-devel qt5 qt5-dbus qt5-declarative-devel qt5-quickcontrols2-devel extra-cmake-modules kconfig-devel kcoreaddons-devel kfilemetadata5-devel ki18n-devel kiconthemes-devel kio-devel kirigami2-devel kxmlgui-devel kdoctools-devel breeze AppStream gettext-devel"
```
Comment (classabbyamp) [inline srcpkgs/haruna/template:8]: wrap at ~80 characters please, and indent the continuation with a single space

File: srcpkgs/haruna/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'haruna'
+
+pkgname=haruna
+version=0.8.0
+revision=1
+build_style=cmake
+hostmakedepends="cmake"
```
Comment (classabbyamp) [inline srcpkgs/haruna/template:7]: implied by the build style

**Comment by classabbyamp** [comment]
please squash into one commit and force-push

**Comment by dragon-friend** [comment]
> please squash into one commit and force-push

thanks for the input!

**Pull request closed without merging**